boot/seal.go

```diff
@@ -45,8 +45,25 @@ var (
 	secbootResealKeys = secboot.ResealKeys
 
 	seedReadSystemEssential = seed.ReadSystemEssential
+
+	// HasFdeSetupHook,FdeSetupHookRunner will be set by the
+	// devicestate code
```

Comment:
Hook function setup by devicestate to support device-specific full disk encryption implementations.